### ISSUE TITLE
Scalar plot not showing entire range of y value

### ISSUE THREAD

**Issue by ZhongBaby** [tensorflow/tensorboard#273]
As you can see, I have a orange curve corresponding to training set loss, and blue one is validation set loss. When I choose both of them, tensorboard only shows the [0, 0.4] range of y value, but I want to see them both.
![1](https://user-images.githubusercontent.com/8222739/28650974-b1e3ef22-72b1-11e7-8ad4-581c6e8d4cf2.PNG)
![2](https://user-images.githubusercontent.com/8222739/28650976-b32d6fd4-72b1-11e7-944e-8bf844fa2154.PNG)

**Comment by chihuahua** [comment]
On the top left of the scalars dashboard, do you see a toggle that reads "Ignore outliers in chart scaling"?

![035ncv93vbo](https://user-images.githubusercontent.com/4221553/28651246-88a1a590-7235-11e7-8744-9b1500da2e63.png)
What happens if you toggle that off?

**Comment by ZhongBaby** [comment]
Thanks!

**Comment by jdhao** [comment]
For those who are seeing very small losses like 0.07, you may see a very flat loss curve because the y-axis data range is like -1 to 1.
I found that disable the `Ignore outliers in chart scaling` works for me. After disabling it, the y-axis is range is changed to reflect the real loss range.

**Comment by fuji321** [comment]
邮件已收到
